package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "openexp",
-  "version": "0.1.2",
+  "version": "0.1.3",
   "description": "An open-source mean stack app for running experiments with openbci",
```

Comment:
ya, we can take that out. even if it can't support other hardware right now, you can still run behavioral exps without the openbci gear